containerless/rust/integration-tests/src/test_runner.rs

```diff
@@ -129,51 +112,31 @@ pub async fn run_test_async(name: &str, js_code: &str, js_requests: Vec<(&str, J
 
     // Create the function in Containerless
     if let Err(err) = runner.containerless_create(name, js_code).await {
-        fail_and_delete(runner, name, err).await;
-        return results;
+        assert!(false, format!("{:?}", err));
     }
 
     // Send requests to the containerized version
     for req in js_requests.into_iter() {
-        match runner.containerless_invoke(name, req).await {
-            Ok(result) => results.push(result),
-            Err(err) => {
-                fail_and_delete(runner, name, err).await;
-                return results;
-            }
-        }
+        let result = runner.containerless_invoke(name, req).await
+            .expect("invoke JavaScript failed");
```

Comment:
We won't want to use expect here because we need to catch any errors and delete the error'd function before the runner returns.

Reply:
Well I guess the exclusive thing takes care of it anyway.